tasks/configure.yml

```diff
@@ -4,19 +4,8 @@
   become: yes
   lineinfile:
     dest: /etc/hosts
-    line: "127.0.0.1 {{ ansible_hostname }}"
-
-- name: Create various directories
-  become: yes
-  file:
-    group: "{{ gocd_server.group }}"
-    owner: "{{ gocd_server.user }}"
-    path: "/home/{{ gocd_server.user }}/{{ item }}"
-    state: directory
-  with_items:
-    - work/go-server
-    - bin
-    - .aws
+    line: "127.0.0.1 localhost {{ ansible_hostname }}"
```

Comment:
It does, by stopping duplicate 127.0.0.1 entries, which keeps the host file cleaner. Why would we drift from the AWS hostnames? I always set a hostname on providers that do not give a default hostname, but I see no reason to do that on AWS.